spec.emu

```diff
@@ -234,6 +236,8 @@ contributors: Ujjwal Sharma, Younies Mahmoud
             1. Throw a *RangeError* exception.
           1. If _unit_ is *"minutes"* or *"seconds"*, then
             1. Set _style_ to *"2-digit"*.
+        1. If _unit_ is *"hours"* and _durationFormat_.[[TwoDigitHours]] is *true*, then
+          1. Set _style_ to *"2-digit"*.
```

Comment:
nit: it could be merged with the condition above
```suggestion
          1. If _unit_ is *"minutes"* or *"seconds"* or _unit_ is *"hours"* and _durationFormat_.[[TwoDigitHours]] is *true*, then
          1. Set _style_ to *"2-digit"*.
```

Reply:
My thought is to leave this one as-is, because stringing together four conditionals on one line will likely be annoying for implementers